src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -615,6 +615,20 @@ public function testResolveEnvValues()
         unset($_ENV['DUMMY_ENV_VAR'], $_SERVER['DUMMY_SERVER_VAR'], $_SERVER['HTTP_DUMMY_VAR']);
     }
 
+    public function testResolveEnvValuesWithArray()
```

Comment:
risky test btw.. needs an assertion.

Reply:
You definitely need to assert something.

Reply:
I did some assertions. Sorry for that.